composer.json

```diff
@@ -1,14 +1,21 @@
 {
-	"name": "ilias/deps",
-	"description": "Dependencies of ILIAS",
+	"name": "ilias",
```

Comment:
This should include a vendor and package name, so `ilias/ilias` would be recommended (https://getcomposer.org/doc/04-schema.md#name)

Reply:
@clue Thanks for the hint. Fixed.